__tests__/utils.test.js

```diff
@@ -24,6 +30,19 @@ describe('unit tests for utility functions', () => {
     expect(winner[1]).toBe(14);
   });
 
+/* jest allows you to organize the functions you are testing by nesting describe calls
+  for example:
+
+  desicribe("checkForDuplicates", () => {
+    it('returns 0 if there are no duplicates in the input array', () => {
+      const input = [8, 10, 14, 6];
+      const dups = checkForDuplicate(input);
+      expect(dups).toBe(0);
+    });
+  });
+*/
+  })
+
```

Comment:
Instead of having one 'describe' call for all of the functions - would a better format be having a 'describe' for every function tested with multiple 'it' calls within it?

Reply:
ya, usually that's more readable.